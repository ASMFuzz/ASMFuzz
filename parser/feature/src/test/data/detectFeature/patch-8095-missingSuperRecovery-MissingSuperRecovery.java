@@ -5,7 +5,7 @@
  * class is no longer available during a subsequent compilation.
  * @author maddox
  * @build impl
- * @compile/fail/ref=MissingSuperRecovery.out --diags=layout=%b:%l:%_%m MissingSuperRecovery.java
+ * @compile/fail/ref=MissingSuperRecovery.out -XDrawDiagnostics MissingSuperRecovery.java
  */
 
 // Requires "golden" class file 'impl.class', which contains