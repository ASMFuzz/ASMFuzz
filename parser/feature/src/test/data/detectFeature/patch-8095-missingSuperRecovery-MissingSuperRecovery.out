@@ -1,5 +1,2 @@
-MissingSuperRecovery.java:15: cannot access base
-public class MissingSuperRecovery extends impl {
-       ^
-  class file for base not found
+MissingSuperRecovery.java:15:43: compiler.err.cant.access: base, (compiler.misc.class.file.not.found: base)
 1 error